tasks/pull_date_format.js

```diff
@@ -0,0 +1,169 @@
+var path = require('path');
+var fs = require('fs');
+var constants = require('./util/constants');
+var common = require('./util/common');
+
+var args = process.argv.slice(2);
+var argLocale = args[0];
+
+var pathToEn = path.join(constants.pathToLib, 'locale-en.js');
+var pathToWCRegions = path.join(__dirname, '../node_modules/world-calendars/dist/regional');
+
+if(!argLocale) {
+    fs.readdir(pathToWCRegions, function(err, items) {
+        if(err) throw err;
+
+        items.forEach(function(item) {
+            var itemLocale = item.split('.')[0];
+            if(itemLocale.substr(0, 2) === 'en') {
+                console.log('skipping English ' + itemLocale);
+                return;
+            }
+            pullOneLocale(itemLocale);
+        });
+    });
+}
+else {
+    pullOneLocale(argLocale);
+}
+
+function pullOneLocale(locale) {
+    var pathToInput = path.join(pathToWCRegions, locale + '.js');
+    var pathToOutput = path.join(constants.pathToLib, 'locale-' + locale.toLowerCase() + '.js');
```

Comment:
I wonder if putting all the locales in a new subdirectory `lib/locales/` (and `dist/locales/`) would make things more user-friendly?

Reply:
happy to make `locales/` directories if you think it's better. Definitely seems nice for `lib/`, but how would it play with the CDN? [Sync](https://github.com/plotly/streambed/blob/master/bin/update_plotlyjs_cdn.sh) probably has to change, is it OK with serving from subdirectories?

Reply:
update from private convo:

![image](https://user-images.githubusercontent.com/6675409/34121262-6880e69e-e3f6-11e7-8284-b7af389fcb1f.png)

Reply:
lib/locale -> https://github.com/plotly/plotly.js/pull/2207/commits/614d5befb50a6b95774fbc04b5fd58596b5cb66d